### PR TITLE
Mbrewer dev bundle

### PR DESCRIPTION
**Pull request by multilinear**
Port from s3cfg to awscli.

   This serves 3 purposes
    1) aws cli can do a lot more things, so this is fewer tools we have to use
    2) this uses the same key format as we're already using for everything else
    3) because of key format, we can easily switch to everyone using their own keys
    
    Note that after this is committed you will have to set up aws cli to use this script.
    See: https://mdg.hackpad.com/Operational-Suggestions-and-tips-EZFrKK81iHf#:h=AWS-management

### CODE REVIEW

File: scripts/admin/copy-dev-bundle-from-jenkins.sh [hunk 1]
```diff
@@ -35,17 +35,17 @@ echo Found build $DIRNAME
...
-s3cmd ls s3://com.meteor.jenkins/$DIRNAME/ | \
+aws s3 ls s3://com.meteor.jenkins/$DIRNAME/ | \
   perl -nle 'if (/\.tar\.gz/) { ++$TAR } else { die "something weird" }  END { exit !($TAR == 4) }'
 
 trap - EXIT
 
-for FILE in $(s3cmd ls s3://com.meteor.jenkins/$DIRNAME/ | perl -nlaF/ -e 'print $F[-1]'); do
-   if s3cmd info $TARGET$FILE >/dev/null 2>&1; then
+for FILE in $(aws s3 ls s3://com.meteor.jenkins/$DIRNAME/ | perl -nlaF/ -e 'print $F[-1]'); do
+   if [[ $(aws s3 ls $TARGET$FILE >/dev/null | wc -l) != 0 ]]; then
```
Comment (Slava) [inline scripts/admin/copy-dev-bundle-from-jenkins.sh]: Why is this change to `wc -l`? Does `aws s3 ls` not exit with `1` on failure?

Comment (multilinear) [inline scripts/admin/copy-dev-bundle-from-jenkins.sh]: Yup.. exactly.

**Comment by multilinear** [comment]
Heads up @stubailo, this would likely affect you as well.

**Comment by stubailo** [comment]
Cool - I'll probably run it, watch it fail, and then come here and set up aws cli

File: scripts/admin/copy-dev-bundle-from-jenkins.sh (hunk 1 above)
Comment (Slava) [inline scripts/admin/copy-dev-bundle-from-jenkins.sh]: I vote for a comment unless it is a wide-spread behavior of cli tools. `grep` for example always returns `1` when the output is empty.

Comment (multilinear) [inline scripts/admin/copy-dev-bundle-from-jenkins.sh]: *shrug* sure. 
It's widespread in the sense that I think aws cli is conistant across all it's calls, but return codes aren't really that standardized in unix. You'd expect it to act like "ls" which it doesn't quite.
A comment sounds reasonable, added.

**Comment by multilinear** [comment]
Merging this isn't worth it... everything changed and rebase or merge is causing tons of conflicts.
I'm abandoning this branch and starting over

**Comment by multilinear** [comment]
... Aaand, I learn that I am not very good at git, I was trying to rebase on master.

**Comment by multilinear** [comment]
@slava any thing else, or can I submit this?

**Comment by Slava** [comment]
I am fine with this change

**Pull request merged**